### PR TITLE
(imp) [SD-2219] Publish Queue Page Improvements

### PR DESCRIPTION
**Pull request by sivakuna-aap**
Correcting pubstatus values as per schema. Also taking care of existing data.

### CODE REVIEW

**Comment by amagdas** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

File: server/apps/archive/archive.py [hunk 1]
```diff
@@ -143,6 +143,15 @@ class ArchiveService(BaseService):
...
     mediaService = ArchiveMediaService()
 
+    def on_fetched(self, docs):
+        """
+        Overriding this to handle existing data in Mongo & Elastic
+        """
+
+        for item in docs[config.ITEMS]:
+            if 'pubstatus' in item:
+                item['pubstatus'] = item.get('pubstatus', 'Usable').capitalize()
```
Comment (mdhaman) [inline server/apps/archive/archive.py]: :+1: 
Can we change to schema to have pubstatus as lowercase (http://cv.iptc.org/newscodes/pubstatusg2/usable)? 
We can easily handle string manipulation on output as well as css.

Comment (sivakuna-aap) [inline server/apps/archive/archive.py]: I will change the values as per IPTC Standards. However, this piece of code handles existing data not for display purpose.

Comment (sivakuna-aap) [inline server/apps/archive/archive.py]: Fixed in https://github.com/superdesk/superdesk/pull/552

**Pull request merged**